docs/modules/operation/pages/database-reports/database.adoc

```diff
@@ -174,9 +174,9 @@ The official Jaspersoft Studio documentation and wiki describe how to do this.
 === Use Measurements Datasource and Helpers
 
 To use the Measurements API you must add the Measurements Datasource library to the build path of JasperStudio.
-To do so, right click in the Project Explorer and select *Configure Buildpath*.
```

Comment:
Does this file's changes belong in this PR?

Reply:
It isn't incorrect, so we can keep it.